test/replication-luatest/join_test.lua

```diff
@@ -64,3 +64,46 @@ g.test_fetch_self_delete_during_final_join = function(lg)
         box.space._cluster:delete{replica_id}
     end, {replica_id})
 end
+
+--
+-- gh-10089: raft state is never sent during META_JOIN stage.
```

Comment:
`is` -> `was`. To make the emphasis that now it is not so, and that is good.

Reply:
Fixed